analysed-packages/libestr/version-0.1.11/libestr-0.1.11-OSS-disclosure.txt

```diff
@@ -0,0 +1,3059 @@
+LIBRARY IS WITH YOU. SHOULD THE LIBRARY PROVE DEFECTIVE, YOU ASSUME THE COST OF
+ALL NECESSARY SERVICING, REPAIR OR CORRECTION.
+
+16. IN NO EVENT UNLESS REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING WILL
+ANY COPYRIGHT HOLDER, OR ANY OTHER PARTY WHO MAY MODIFY AND/OR REDISTRIBUTE THE
+LIBRARY AS PERMITTED ABOVE, BE LIABLE TO YOU FOR DAMAGES, INCLUDING ANY GENERAL,
+SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY
+TO USE THE LIBRARY (INCLUDING BUT NOT LIMITED TO LOSS OF DATA OR DATA BEING
+RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD PARTIES OR A FAILURE OF
+THE LIBRARY TO OPERATE WITH ANY OTHER SOFTWARE), EVEN IF SUCH HOLDER OR OTHER
+PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
+
+END OF TERMS AND CONDITIONS
+
+How to Apply These Terms to Your New Libraries
+
+If you develop a new library, and you want it to be of the greatest possible use
+to the public, we recommend making it free software that everyone can
+redistribute and change. You can do so by permitting redistribution under these
+terms (or, alternatively, under the terms of the ordinary General Public
+License).
+
+To apply these terms, attach the following notices to the library. It is safest
+to attach them to the start of each source file to most effectively convey the
+exclusion of warranty; and each file should have at least the "copyright" line
+and a pointer to where the full notice is found.
+
+     one line to give the library's name and an idea of what it does.
+     Copyright (C) year  name of author
+
+     This library is free software; you can redistribute it and/or modify it
+under the terms of the GNU Lesser General Public License as published by the
+Free Software Foundation; either version 2.1 of the License, or (at your option)
+any later version.
+
+     This library is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+details.
+
+     You should have received a copy of the GNU Lesser General Public License
+along with this library; if not, write to the Free Software Foundation, Inc., 51
+Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA Also add information
+on how to contact you by electronic and paper mail.
+
+You should also get your employer (if you work as a programmer) or your school,
+if any, to sign a "copyright disclaimer" for the library, if necessary. Here is
+a sample; alter the names:
+
+Yoyodyne, Inc., hereby disclaims all copyright interest in
+the library `Frob' (a library for tweaking knobs) written
+by James Random Hacker.
+
+signature of Ty Coon, 1 April 1990
+Ty Coon, President of Vice
+That's all there is to it!
+
+--------------------------------------------------------------------------------
+
+LicenseRef-fossology-GPL-3.0-or-later WITH Autoconf-exception-generic-3.0
```

Comment:
**libestr-0.1.11-OSS-disclosure.txt** file is adapted as you suggested. Thank you!

I have the same name also in spdx file. It seems our fossology server will produce these prefixes. I have informed my colleagues. In the meanwhile, I have adapted them manually. It seems that I should change **LicenseRef-fossology-GPL-3.0-or-later WITH Autoconf-exception-generic-3.0** to **LicenseRef-GPL-3.0-or-later WITH Autoconf-exception-generic-3.0** in spdx files.  So LicenseRef stays and only fossology part is removed. Otherwise I get error during conversion of spdx files. Warning for this case, 
![Screenshot from 2024-04-29 17-17-23](https://github.com/Open-Source-Compliance/package-analysis/assets/130649329/704c0363-2f62-4133-b171-3157e6436d50)

And an actual error in case, I remove the whole **LicenseRef-fossology-** from license Id. 

Although if I just chage this prefix from **LicenseRef-fossology-**  to **LicenseRef-** then it works properly.

![Screenshot from 2024-04-29 17-06-53](https://github.com/Open-Source-Compliance/package-analysis/assets/130649329/a3bac735-1f34-4660-aabc-8f4d1c948e59)

Reply:
@ParianGol ah I see I was not precise, sorry for this.
Sure the "LicenseRef..." must stay in the SPDX files as prefix for the LicenseIDs, otherwise the SPDX files will not be valid. I ment to remove it from the license name printed in the Disclosure document